collections/open-journalism/index.md

```diff
@@ -0,0 +1,6 @@
+---
+items: fivethirtyeight/data, NYTimes/objective-c-style-guide, newsapps/beeswithmachineguns, voxmedia/meme, propublica/guides, censusreporter/censusreporter, nprapps/app-template, TimeMagazine/babynames, guardian/frontend, dukechronicle/chronline, BloombergMedia/whatiscode, times/cardkit
+display_name: Open journalism
+created_by: benbalter
```

Comment:
Reading some of these collections, I feel like some automated tests should be added, either in a separate branch that lands first or in this branch, to keep each collection in the right format. Some things I can think of to check:

- `items` list in the right format and contains items that we support, like YouTube URLs, repository `owner/name` strings, and GitHub organizations.
- `display_name`, body, and `items` are present.
- Oxford comma test and similar formatting checks like what we use for topic bodies, but for collection bodies.
- Images are the right size and a valid format.
- `image` field has the same file name as the image in the collection's directory.